overlay.d/07c9s/bootstrap/pre-pivot.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script is called by the OKD installer and runs in the
+# local context of the bootstrap node.
+# It rebases the booted operating system (e.g. Fedora CoreOS)
+# to CentOS Stream CoreOS 9
+
+# Load common functions
+. /usr/local/bin/bootstrap-service-record.sh
+. /usr/local/bin/release-image.sh
+
+record_service_stage_start "rebase-to-scos"
+
+# Copy manifests
+# Before rebasing an FCOS node to SCOS, ensure the OKD manifests are in place
+if [ ! -d /opt/openshift/openshift/ ]; then
+    mkdir -p /opt/openshift/openshift/
+fi
+cp -irvf manifests/* /opt/openshift/openshift/
+
+# Disable all yum repos
+sed -i 's/^enabled=1/enabled=0/g' /etc/yum.repos.d/*.repo
+
+# Pivot to new ostree container image
+OS_IMAGE=$(image_for centos-stream-coreos-9)
```

Comment:
Note: This image doesn't exist yet in CI or any payloads. It'll be created with https://github.com/okd-project/okd-coreos-pipeline on operate-first.cloud, pushed to quay.io and imported into CI from there. A new release controller config will then be set up to create OKD/SCOS (CI) release payloads that include the image.